common/src/main/webapp/sass/explore.scss

```diff
@@ -157,7 +161,212 @@ $tree-section-width: 300px;
       display: unset;
     }
   }
+}
+
+// ------
+// Sidereal Target Editor
+// ------
+.explore-brightnesses-wrapper {
+  grid-column: 1 / -1;
+  justify-self: stretch;
+  align-self: stretch;
+}
+
+.explore-brightnesses-container {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.explore-brightnesses-units-dropdown.p-dropdown {
+  white-space: nowrap;
+  width: 100%;
+}
+
+.explore-brightnesses-delete-button-wrapper {
+  display: flex;
+  justify-content: center;
+
+  .p-button.delete-button {
+    padding: 0;
+    flex-shrink: 0;
+  }
+}
+
+.target-grid {
+  display: grid;
+  grid-gap: 0.5em;
+  grid-row-gap: 0.3em;
+  padding: 0.5em;
+  height: 100%;
+
+  .explore-title-undo-buttons {
+    grid-area: undo;
+  }
+
+  .target-aladin-cell {
+    grid-area: aladin;
+  }
+
+  .target-form {
+    grid-area: coordinates;
+  }
+
+  .target-proper-motion-form {
+    grid-area: proper-motion;
+
+    .p-inputgroup>span:first-of-type {
+      width: 4em;
+    }
+  }
+
+  .target-source-profile-editor {
+    grid-area: source-profile;
+  }
+}
+
+.target-rv-controls {
+  justify-self: stretch;
+
+  .explore-grow-1 {
+    margin-left: 0.3em !important;
+  }
+}
+
+@mixin expanded-aladin {
+  grid-template-areas: 'aladin';
+  grid-template-columns: 1fr;
+
+  .explore-title-undo-buttons {
+    display: none;
+  }
 
+  .target-form {
+    display: none;
+  }
+
+  .target-proper-motion-form {
+    display: none;
+  }
+
+  .target-source-profile-editor {
+    display: none;
+  }
+}
+
+.tile-xl-width,
+.tile-lg-width,
+.tile-md-width,
+.tile-sm-width {
+  .target-grid {
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: min-content min-content min-content 1fr;
+    grid-template-areas:
+      'undo           aladin'
+      'coordinates    aladin'
+      'proper-motion  aladin'
+      'source-profile aladin';
```

Comment:
It seems the layout was different for the different sizes at one point? This grid layout provides more opportunities for changing the layout. However, currently, either everything is shown, or only `aladin` is shown (small screens, full-screen aladin). If we want to keep this behavior, I could simplify this grid layout and remove some of the commented out CSS below.

Reply:
Right. I think we wanted to show aladin and hide the lesser parts of the form on small screen. Essentially show the coordinates only and below aladin in full width.
This is worth testing in several configurations and also on full screen mode

Reply:
OK. Currently we do NOT show the coordinates on small screens. (The `expanded-aladin` mixin above is used for both full screen aladin and small screens.) I can change it so that the coordinates are displayed above (or below) aladin for small screens if that is what we want.

Reply:
Did we only show the name before?
I guess coordinates is often usefil

Reply:
At this time (in staging and production) we don't show the name. We only show aladin. At some point maybe we showed the name and/or coordinates? If we show the coordinates, we probably want to show the undo/redo buttons, too, I guess?

Reply:
Ok, and we are showing the table right? In that case the coords are there

Reply:
Ah. OK. Do you want to be able to edit the name and coords? I pushed a change that show them above aladin. But I could remove them again. They can still go to full screen aladin in the small screen, so 🤷‍♂️

Reply:
I'd keep as i twas before

Reply:
Which is to not allow editing in small screen and only show aladin?